client/components/term-picker-option/style.scss

```diff
@@ -0,0 +1,77 @@
+.term-picker-option {
+  $padding-size: 18px;
+
+  display: flex;
+  width: 100%;
+  cursor: pointer;
+
+  padding: $padding-size;
+  border-radius: 5px;
+
+  background: #FFF;
+
+  .term-picker-option__radio-wrapper {
+    width: 21px;
+    margin-right: 12px;
+  }
+
+  .term-picker-option__radio {
+    width: 21px;
+    height: 21px;
+
+    &:checked:before {
+      width: 13px;
+      height: 13px;
+    }
+  }
+
+  .term-picker-option__header,
+  .term-picker-option__description {
+    width: 100%;
+  }
+
+  .term-picker-option__term,
+  .term-picker-option__price,
+  .term-picker-option__discount,
+  .term-picker-option__side-note,
+  .term-picker-option__description {
+    display: inline-block;
+    vertical-align: middle;
+  }
+
+  .term-picker-option__header {
+    line-height: 24px;
+  }
+
+  .term-picker-option__term {
+    display: inline-block;
+    font-size: 24px;
+    line-height: 24px;
+  }
+
+  .term-picker-option__discount {
+    margin-left: 10px;
+  }
+
+  .term-picker-option__description {
+    font-size: 16px;
+  }
+
+  .term-picker-option__price {
+    color: #4f646e;
+  }
+
+  .term-picker-option__side-note {
+    color: #59849D;
+    margin-left: 5px;
+  }
+
+  &:not( .term-picker-option--active ) {
+    border: 1px solid #c5d8e2;
+    padding: $padding-size + 2px;
```

Comment:
I'm confused why you've defined a variable, and then only use it once — and adding to it. Seems like the variable isn't really necessary at all?

Reply:
It's actually used twice, here and in `.term-picker-option` definition